### PR TITLE
Trouble ticket

### PR DESCRIPTION
**Pull request by lukecampbell**

### CODE REVIEW

**Comment by birdage** [comment]
@lukecampbell are these imagery free and available for use?

**Comment by lukecampbell** [comment]
I believe so, I'd have to ask Dan. I can't find it on Google.

**Comment by birdage** [comment]
@dlangston any comment on this?

**Comment by dlangston** [comment]
Should be good to go.

**Comment by dlangston** [comment]
now it's good

**Comment by DanielJMaher** [comment]
@birdage Can you review so I can work this?

File: ooiui/templates/common/troubleTicket.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  </div><!-- =page-content-wrapper -->
+
+</div> <!-- wrapper -->
+
+
+<script type="text/javascript">
+_.extend(OOI.prototype, Backbone.Events, {
+  login: new LoginModel(),
+  views: {
+    ticket: new TroubleTicketView()
```
Comment (lukecampbell) [inline ooiui/templates/common/troubleTicket.html]: @DanielJMaher You need to insert a trouble ticket model here. 
```javascript
ticket: new TroubleTicketView({model: new TroubleTicketModel()});
```

File: ooiui/static/js/views/common/TroubleTicketView.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"use strict";
+/*
+* ooiui/static/js/views/common/TroubleTicketView.js
+*
+* Dependencies
+* Partials
+* - ooiui/static/js/partials/TroubleTicket.html
+*/
+var TroubleTicketView = Backbone.View.extend({
```
Comment (lukecampbell) [inline ooiui/static/js/views/common/TroubleTicketView.js]: @DanielJMaher you'll need an event handler here to capture the button click and save the model.

```
events: {
    'click .btn-sm' : 'onSubmit'
}
```

File: ooiui/static/js/views/common/TroubleTicketView.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  initialize: function() {
+    _.bindAll(this, "render");
+    console.log("binded");
+    this.render();
+  },
```
Comment (lukecampbell) [inline ooiui/static/js/views/common/TroubleTicketView.js]: @DanielJMaher You'll need a submit method that get's the form field data, puts it in the model and then saves the model.

Comment (lukecampbell) [inline ooiui/static/js/views/common/TroubleTicketView.js]: I'd strongly encourage you to exploit stickit here.
http://jsfiddle.net/nareshbhatia/jjdm5/

**Pull request merged**